feature-libs/product-configurator/rulebased/components/price/configurator-price.component.ts

```diff
@@ -54,6 +62,14 @@ export class ConfiguratorPriceComponent {
     return quantity ? this.formula.price?.value !== 0 && quantity >= 1 : false;
   }
 
+  /**
+   * Verifies if 'Selected' should be shown
+   *
+   * @return {boolean} - 'true' if 'Selected' should be shown, otherwise 'false'
+   */
+  displaySelected(): boolean {
```

Comment:
Plz, remove this method. It is not required.

Reply:
done